docs/SUMMARY.md

```diff
@@ -16,6 +16,7 @@
   - [Metadata](features/metadata.md)
   - [Transactions/Export](features/transactions/export.md)
   - [Transactions/Search](features/transactions/search.md)
+  - [Protocols/Bitcoin](features/protocols/bitcoin.md)
```

Comment:
I'm just thinking, if something like `URI Protocols`/`URI Custom Protocols` would be better naming. 🤔 Not only in this place... but no strong opinion here.

Reply:
I agree but I will leave it to a follow-up issue. It would be really annoying to commit all the changes to correct fixups and do not break anything.

Reply:
👌